AndriyZhornyak/HMW4/HMW4.py

```diff
@@ -0,0 +1,21 @@
+# #task 1
```

Comment:
good job
I would advise you to use meaningful names for variables

celsius = float(input("Enter the temperature in Celsius: "))


if celsius < -273.15:
    print("Error: Temperature below absolute zero (-273.15°C)")
else:
    
    fahrenheit = (celsius * 9/5) + 32

   
    print("{:.1f}°C is equivalent to {:.1f}°F".format(celsius, fahrenheit))